test/libsolidity/syntaxTests/natspec/docstring_private_state_variable.sol

```diff
@@ -0,0 +1,7 @@
+contract C {
+    /// @notice example of notice
+    /// @dev example of dev
+    uint private state;
+}
+// ----
+// Warning: Documentation tag on non-public state variables will be disallowed in 0.7.0. You will need to use the @dev tag explicitly.
```

Comment:
It is just the "notice" tag that will be disallowed, right? Can you change the text to mention that `@dev` can still be used?